.run/Production.run.xml

```diff
@@ -1,7 +1,9 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Production" type="FlutterRunConfigurationType" factoryName="Flutter">
+    <option name="additionalArgs" value="--no-sound-null-safety" />
+    <option name="attachArgs" value="--no-sound-null-safety" />
     <option name="buildFlavor" value="production" />
     <option name="filePath" value="$PROJECT_DIR$/lib/main-production.dart" />
     <method v="2" />
   </configuration>
-</component>
+</component>
```

Comment:
man...just fix the AS setup to avoid this same issue reoccurs...I believe it shouldn't trim it like this...
![Screen Shot 2564-04-09 at 17 17 21](https://user-images.githubusercontent.com/13435717/114166198-a129ae80-9957-11eb-9bcd-3837096f5b46.png)


Also, I think you may want to gitignore this somehow? 🤔  this is your own Run setup right?

Reply:
https://stackoverflow.com/questions/24093609/how-can-i-configure-android-studio-to-add-new-line-at-end-of-file#:~:text=Yes%2C%20you%20can%20configure%20Android,the%20setting%20has%20been%20enabled.

Reply:
> man...just fix the AS setup to avoid this same issue reoccurs...I believe it shouldn't trim it like this...
I did tick it in Android Studio, but it doesn't affect these files 😁 

> Also, I think you may want to gitignore this somehow? 🤔 this is your own Run setup right?
I want to share it, so other peoples can use these configs to run & build project